### ISSUE TITLE
App crashes when trying look at someone's user profile

### ISSUE THREAD

**Issue by NurMarvin** [replugged-org/replugged#198]
**Describe the bug**
Trying to open someone's user profile crashes the app.

**To Reproduce**
1. Open someone's full user profile (the modal)
2. Observe

**Expected behavior**
App doesn't crash

**Desktop (please complete the following information):**
 - OS: Windows 11, Version 21H2 (Build 22000.978)
 - Replugged version: 30fd95a

**Additional context**
 - [x] I checked if Discord and/or Replugged are up to date
 - [x] I made sure this is not related to an external plugin or a theme
 
**Debug Information**
```ini
# Debugging Information | Result created: Fri, 16 Sep 2022 19:30:57 GMT
[SYSTEM / DISCORD]
Locale="en-US"
OS="Windows 64-bit"
Architecture="x64"

ReleaseChannel="canary"
AppVersion="1.0.49"
BuildNumber="147550"
BuildID="ff400d2"
Experiments="11/159"

[PROCESS VERSIONS]
React="17.0.2"
Electron="13.6.6"
Chrome="91.0.4472.164"
Node="14.16.0"

[REPLUGGED]
Commands="12"
Settings="7"
Plugins="0/5"
Themes="0/0"
Labs="1/2"
SettingsSync="true"
CachedFiles="805"
Account="true"
APIs="9"

[GIT]
Upstream="Official"
Revision="30fd95a"
Branch="main"
Latest="true"

[LISTINGS]
RepluggedPath="C:\Users\U********\Documents\powercord"
DiscordPath="C:\Users\U********\AppData\Local\DiscordCanary\app-1.0.49"
Experiments="2022-06_bot_default_commands_scope, 2022-08_amoled_mode_web, 2021-10_desktop_bug_reporter_enabled, 2022-08_message_todos_staff_only, 2022-06_marketplaces_guild, 2021-02_staff_qc_debugging, 2022-04_donut_experiment, 2022-06_profile_skins, 2022-03_discovery_guild_seo, 2022-09_automod_keyword_regex, 2022-08_webauthn"
Labs="Plugin Embeds"
Plugins=""
```

**Comment by bop34** [comment]
Have you tried disabling experiments?

**Comment by elementemerald** [comment]
Experiencing this issue as well. Nothing to do with experiments, afaik.

**Comment by NurMarvin** [comment]
> Have you tried disabling experiments?

Yes. Issue is still persistent.

**Comment by DuroCodes** [comment]
Experiencing the same issue on macOS

**Comment by asportnoy** [comment]
This was fixed by #199
